### PR TITLE
Update DETableHeader colors and update hover/select theme default

### PR DESCRIPTION
**Pull request by ash3rz**
This will update the DETableHeader to the default colors (not blue) to better match the prototype:

image

I also realized with looking into theming more that we don't need a custom `TableRow` component, we just need to update the theme's opacity for hover and select.  Here's the before and after in a table:

Before - top selected, 2nd is hovered:
image

After - top is selected, 2nd is hovered:
image

At least for me, the before is way too light to see very well.  We can obviously tweak those numbers too, but this didn't look too bad to me.

I also realized all the tests were broken so I fixed those.  I'm not sure why we were importing from `lib` instead of `material-ui` directly 🤷‍♀

### CODE REVIEW

**Review by sri2k1us — APPROVED** [review]
LGTM :shipit:

File: src/__tests__/tableHeader.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 import ReactDOM from "react-dom";
+import { MuiThemeProvider } from "@material-ui/core/styles";
 
-import { getMuiTheme, MuiThemeProvider } from "../lib";
-import { DETableRowTest } from "../../stories/DETableRow.stories";
+import { getMuiTheme } from "../lib";
+import { DETableHeader } from "../../stories/TableHeader.stories";
 
-it("renders DETableRow without crashing", () => {
+it("renders trigger field without crashing", () => {
```
Comment (psarando) [inline src/__tests__/tableHeader.js]: "renders Table Header"?

File: src/util/table/EnhancedTableHead.js [hunk 2]
```diff
@@ -48,15 +45,14 @@ class EnhancedTableHead extends React.Component {
...
-                            <Checkbox
+                        <TableCell padding={padding ? padding : "checkbox"}>
+                            <DECheckbox
                                 indeterminate={isInDeterminate}
                                 checked={numSelected === rowsInPage}
                                 onChange={onSelectAllClick}
-                                className={classes.column_heading}
+                                inputProps={{
+                                    "aria-label": getMessage("selectAll"),
+                                }}
```
Comment (psarando) [inline src/util/table/EnhancedTableHead.js:55]: 👍

**Review by psarando — APPROVED** [review]
LGTM 👍

File: src/__tests__/tableHeader.js (hunk 1 above)
Comment (ash3rz) [reply src/__tests__/tableHeader.js]: 😊 oops, copy-pasta

**Comment by ash3rz** [comment]
Thanks for the reviews!

**Pull request merged**